ios/Lunes/Info.plist

```diff
@@ -22,6 +22,8 @@
 	<string>????</string>
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
+    <false/>
```

Comment:
Not sure about this one

Reply:
Added this, because otherwise the export complience form must be sent manually every time.

Reply:
Yes, this is correct.